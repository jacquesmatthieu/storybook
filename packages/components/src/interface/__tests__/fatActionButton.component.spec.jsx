import React from 'react';
import { shallow } from 'enzyme';

import FatActionButton from '../fatActionButton.component';

describe('FatActionButton', () => {
  let props;

  beforeEach(() => {
    props = {
      icon: 'plus',
      text: 'Foo',
    };
  });

  const getWrapper = () => shallow(<FatActionButton {...props} />);

  it('should render properly', () => {
    // When
    const wrapper = getWrapper();

    // Then
    expect(wrapper).toMatchSnapshot();
  });
});
