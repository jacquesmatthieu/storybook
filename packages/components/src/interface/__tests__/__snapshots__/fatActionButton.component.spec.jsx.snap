// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FatActionButton should render properly 1`] = `
<fatActionButtoncomponent__FatButton>
  <fatActionButtoncomponent__IconCircle>
    <Icon
      name="plus"
      size={15}
      theme={
        Object {
          "palette": Object {},
        }
      }
    />
  </fatActionButtoncomponent__IconCircle>
  <fatActionButtoncomponent__ButtonText>
    Foo
  </fatActionButtoncomponent__ButtonText>
</fatActionButtoncomponent__FatButton>
`;
